src/review/prompt/constructPrompt.ts

```diff
@@ -1,22 +1,44 @@
+import { instructionPrompt } from "../constants";
 import { readFile } from "fs/promises";
-import { filePromptTemplate, instructionPrompt } from "../constants";
 
-const appendToLastPrompt = (
-  prompts: string[],
-  text: string,
-  maxPromptLength: number
-): string[] => {
-  const newPrompts = [...prompts];
-  const lastPromptIndex = newPrompts.length - 1;
+interface ReviewFile {
+  fileName: string;
+  fileContent: string;
+}
 
-  if ((newPrompts[lastPromptIndex] + text).length > maxPromptLength) {
-    newPrompts.push(instructionPrompt + text);
-  } else {
-    newPrompts[lastPromptIndex] += text;
-  }
+const getSizeOfReviewFile = (file: ReviewFile): number =>
+  file.fileName.length + file.fileContent.length;
 
-  return newPrompts;
-};
+const splitFilesIntoBatches = (
```

Comment:
does this not do the same thing as the splitFilesIntoChunks which I pushed this morning?